cv-basics/blob detection.ipynb

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
<p>place it above the example</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>